src/PendingMove.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+
+public class PendingMove {
```

Comment:
It may be helpful to think of this class like a **form**. We *pass* it with all of the inputs, and it tells us the selected output is (in this case, a `Move`). If it's not a valid set of inputs, it can tell you that too.